### PR TITLE
COMMERCE-11193 | COMMERCE-11256

### PR DESCRIPTION
**Pull request by 22LucaGiannotta22**
Resent from #3864 

https://liferay.atlassian.net/browse/COMMERCE-11193
https://liferay.atlassian.net/browse/COMMERCE-11256

### CODE REVIEW

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase [hunk 1]
```diff
@@ -1172,6 +1172,956 @@ definition {
...
+		task ("Then the bookedQuantityId equals 0 for every order item in the customer order") {
+			TestUtils.assertEquals(
+				actual = ${bookedQuantityIds},
+				expected = "0,0");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-11805. Customer order status doesn't change after an order item in the supplier order is added for a shipment"
+	@priority = 4
+	test CanSeeCustomerOrderStatusUnchangedAfterAnOrderItemIsAddedForShipmentInASupplierOrder {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase:1340]:
```suggestion
	test CanAssertCustomerOrderStatusUnchangedAfterSupplierOrderItemIsAddedToShipment {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase [hunk 2]
```diff
@@ -1172,6 +1172,956 @@ definition {
...
+			AssertElementPresent(
+				key_parameter = "$ 64.00",
+				key_status = "Processing",
+				locator1 = "CommerceOrders#ORDER_STATUS_BY_SECOND_PARAMETER");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-11801. When the admin changes the price of an order line item from a splitted supplier order, the changes are reflected to the order line item of the customer order"
+	@priority = 5
+	test CanSeePriceChangesFromSplittedSupplierOrdersAreReflectedToTheCustomerOrder {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase:1469]:
```suggestion
	test CanAssertCustomerOrderReflectsSupplierOrderItemPriceChanges {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase [hunk 3]
```diff
@@ -1172,6 +1172,956 @@ definition {
...
+				productName = "Simple Product",
+				productQuantity = 1,
+				productSku = "TESTSKU",
+				productTotalPrice = "100.00");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-11754. The admin can see split orders prices taken from customer order"
+	@priority = 5
+	test CanSeePriceOfSplittedSupplierOrdersIsTakenFromCustomerOrder {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase:1590]:
```suggestion
	test CanAssertSupplierOrderPriceIsTakenFromCustomerOrder {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase [hunk 4]
```diff
@@ -1172,6 +1172,956 @@ definition {
...
+					key_cellName2 = "",
+					key_columnNumber = 5,
+					locator1 = "CommerceEntry#TABLE_CELL_BY_COLUMN_ORDER_NO_TITLE");
+			}
+		}
+	}
+
+	@description = "This is a test for COMMERCE-11750. The admin can see that the split supplier orders are linked to the right channel when both catalog and channel are linked to the supplier account"
+	@priority = 5
+	test CanSeeSplitSupplierOrdersLinkedToTheRightChannelWhenBothCatalogAndChannelAreLinkedToSupplierAccount {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase:1701]:
```suggestion
	test CanAssertSupplierOrdersAreLinkedToCorrectChannelsWhenCatalogAndChannelAreLinkedToSupplierAccount {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase [hunk 5]
```diff
@@ -1172,6 +1172,956 @@ definition {
...
+				key_cellName2 = "",
+				key_columnNumber = 4,
+				locator1 = "CommerceEntry#TABLE_CELL_BY_COLUMN_ORDER_NO_TITLE",
+				value1 = "Minium Portal");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-11751. The admin can see that the split supplier orders are linked to the right channel when only the catalog is linked to the supplier account"
+	@priority = 4
+	test CanSeeSplitSupplierOrdersLinkedToTheRightChannelWhenOnlyCatalogIsLinkedToSupplierAccount {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase:1844]:
```suggestion
	test CanAssertSupplierOrdersAreLinkedToCorrectChannelsWhenCatalogIsLinkedToSupplierAccount {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase [hunk 6]
```diff
@@ -1172,6 +1172,956 @@ definition {
...
+				key_cellName2 = "",
+				key_columnNumber = 4,
+				locator1 = "CommerceEntry#TABLE_CELL_BY_COLUMN_ORDER_NO_TITLE",
+				value1 = "Minium Portal");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-11753. The admin can see that the split supplier orders are linked to the right channel when only the channel is linked to the supplier account"
+	@priority = 4
+	test CanSeeSplitSupplierOrdersLinkedToTheRightChannelWhenOnlyChannelIsLinkedToSupplierAccount {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase:1986]:
```suggestion
	test CanAssertSupplierOrdersAreLinkedToCorrectChannelsWhenChannelIsLinkedToSupplierAccount {
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
@22LucaGiannotta22 i added a new commit to the original pr

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:stop

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**